packages/api/lambdas/index-from-database.js

```diff
@@ -42,6 +42,7 @@ async function indexFromDatabase(esIndex, tables, esHost) {
   await Promise.all([
     indexModel(esClient, tables.collectionsTable, esIndex, indexer.indexCollection),
     indexModel(esClient, tables.executionsTable, esIndex, indexer.indexExecution),
+    indexModel(esClient, tables.asyncOperationsTable, esIndex, indexer.indexOperation),
```

Comment:
This is indexing from the database into elasticsearch. I don't think we're using elasticsearch for these?

Reply:
right, this was an old attempt. I'll remove it.